workers/data_refinery_workers/processors/test_salmon.py

```diff
@@ -247,11 +249,6 @@ def test_run_salmon_failure(self):
         job_context = salmon._run_salmon(job_context)
 
         self.assertFalse(job_context["success"])
-        self.assertEqual(processor_job.failure_reason,
-                         ("Shell call to salmon failed because: "
-                          "Error: The index version file missing/versionInfo.json doesn't seem"
-                          " to exist.  Please try re-building the salmon index.]\\nsalmon quant"
-                          " was invoked improperly.\\nFor usage information, try salmon quant "))
```

Comment:
I'll agree that this test is overly specific, but I do think a check to make sure that the `failure_reason` got set to _something_ might be good.